F/FastParzenWindows/Package.toml

```diff
@@ -1,3 +1,3 @@
 name = "FastParzenWindows"
 uuid = "d2256c36-f6cb-4f2b-bb45-9577641c7b75"
-repo = "https://github.com/HITS-AIN/FastParzenWindows.jl.git"
+repo = "https://github.com/ngiann/FastParzenWindows.jl"
```

Comment:
Should still end with `.git`

Reply:
Thanks. Concerning older versions: I am very doubtful that this package is used by anyone else as it has not been advertised.